README.md

```diff
@@ -1,9 +1,442 @@
+***WARNING: HarbourBridge is currently under development, and not all modules
+have been checked in yet. In short, it probably won't compile right now. Please
+check back later.  Or take a look at [pending pull
+requests](https://github.com/cloudspannerecosystem/harbourbridge/pulls).***
+
 # HarbourBridge: Turnkey Postgres-to-Spanner Evaluation
-[![cloudspannerecosystem](https://circleci.com/gh/cloudspannerecosystem/harbourbridge.svg?style=svg)](https://circleci.com/gh/cloudspannerecosystem/harbourbridge)
 
-HarbourBridge is a stand-alone tool for Cloud Spanner evaluation, using data
-from an existing PostgreSQL database. The tool ingests pg_dump output,
-automatically builds a Spanner schema, and creates a new Spanner database
-populated with data from pg_dump.
+HarbourBridge is a stand-alone open-source tool for Cloud Spanner evaluation,
+using data from an existing PostgreSQL database. The tool ingests pg_dump
+output, automatically builds a Spanner schema, and creates a new Spanner
+database populated with data from pg_dump. The tool is designed to simplify
+Spanner evaluation. It is not intended for production database migration. In
+particular, statements and features in the pg_dump output that don't map
+directly on Spanner features are simply ignored. It is also not intended for
+databases larger than a couple of GB.
+
+To use the tool on a PostgreSQL database called mydb, run
+
+```sh
+pg_dump mydb | harbourbridge
+```
+
+HarbourBridge accepts pg_dump's standard plain-text format, but not archive or
+custom formats.
+
+HarbourBridge automatically determines the cloud project and Spanner instance to
+use, and generates a new Spanner database name (prefixed with `pg_dump_` and
+today's date). Command-line flags can be used to explicitly set the Spanner
+instance or database name.
+
+**WARNING: Please check that permissions for the Spanner instance used by
+HarbourBridge are appropriate. Spanner manages access control at the database
+level, and the database created by HarbourBridge will inherit default
+permissions from the instance. All data written by HarbourBridge is visible to
+anyone who can access the created database.**
+
+As it processes the pg_dump data, HarbourBridge reports on progress, provides
+stats on the schema and data conversion steps, and an overall assessment of the
+quality of the conversion. It also generates a schema file and report file (and
+a bad-data file if data was dropped). See
+[Files Generated by HarbourBridge](#files-generated-by-harbourbridge). Details
+of how PostgreSQL is mapped to Spanner can be found in the
+[Schema Conversion](#schema-conversion) section.
+
+This tool is part of the Cloud Spanner Ecosystem, a community contributed and
+supported open-source repository. Please [report
+issues](https://github.com/cloudspannerecosystem/harbourbridge/issues) and send
+pull requests. Note that this tool is not officially supported as part of the
+Cloud Spanner product.
+
```

Comment:
Do you want to link to the whitepaper here?

Reply:
I'll add link when I add the whitepaper.